### PR TITLE
Add tests.py file for reservation app and update models.py

### LINKED ISSUE #53: Add tests for reservation app

### PR DESCRIPTION
**Pull request by idosinger98**
At the time of creating this PR we still don't have the Client and Professional apps. When we add the Client and Professional apps, I'll make the necessary changes in the tests.py and models.py.
Resolve #53

### CODE REVIEW

File: reservation/tests.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+MEETING_TIME = 60
+
+
+@pytest.fixture
+def typeOfJob():
+    return TypeOfJob(professional_id=PROFESSIONAL_ID, typeOfJob_name=TYPEOFJOB_NAME, price=PRICE)
+
+
+@pytest.fixture
+def persisted_typeOfJob_pool(typeOfJob):
```
Comment (Yarboa) [inline reservation/tests.py:25]: the typeOfJob -> consider ranaming to type_of_job

File: reservation/tests.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+
+@pytest.fixture
+def appointment(typeOfJob):
+    return Appointment(professional_id=PROFESSIONAL_ID, client_id=CLIENT_ID, typeOfJob_id=typeOfJob,
+                       start_appointment=START_APPOINTMENT, end_appointment=END_APPOINTMENT, summary=SUMMARY)
+
+
+@pytest.fixture
+def persisted_appointment(db, appointment):
```
Comment (Yarboa) [inline reservation/tests.py:39]: Why do you call this fixture DB? 
Please remove the db here.

db helper already marked the fixture is running from class test methods

File: reservation/tests.py (hunk 1 above)
Comment (idosinger98) [reply reservation/tests.py]: Done

File: reservation/tests.py (hunk 2 above)
Comment (idosinger98) [reply reservation/tests.py]: Done

**Review by Yarboa — CHANGES_REQUESTED** [review]
Please take a look here 
https://github.com/redhat-beyond/Pro4U/blob/main/review/migrations/0002_test_data.py#L29-L33

It does not belong there 

I think you have an error with nested classes also
https://github.com/redhat-beyond/Pro4U/blob/main/review/models.py#L30-L36

I think you can learn of importance of tests now

**Comment by idosinger98** [comment]
> Please take a look here https://github.com/redhat-beyond/Pro4U/blob/main/review/migrations/0002_test_data.py#L29-L33
> 
> It does not belong there
> 
> I think you have an error with nested classes also https://github.com/redhat-beyond/Pro4U/blob/main/review/models.py#L30-L36
> 
> I think you can learn of importance of tests now

I think you got confused with Guy
I did not create these files

**Review by TalReinfeld — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**